src/acceptance/app/app_suite_test.go

```diff
@@ -141,3 +164,47 @@ func averageMemoryUsedByInstance(appGUID string, timeout time.Duration) uint64 {
 
 	return memSum / uint64(len(memoryUsedArray))
 }
+
+func generateDynamicScaleOutPolicy(instanceMin, instanceMax int, threshold int64) string {
+	scalingOutRule := ScalingRule{
+		MetricType:            "memoryused",
+		StatWindowSeconds:     interval,
+		BreachDurationSeconds: interval,
+		Threshold:             int64(threshold),
```

Comment:
good catch, thanks... forget to remove it after I changed the function declaration.  A fix was committed.